lib/cli.py

```diff
@@ -136,13 +155,13 @@ def process(self):
         images_A += images_B.mean(axis=(0, 1, 2)) - \
             images_A.mean(axis=(0, 1, 2))
 
-        print('press "q" to stop training and save model')
+        print('Press "q" to stop training and save model')
 
         BATCH_SIZE = 64
 
-        for epoch in range(1000000):
+        for epoch in range(1, 1000000):
             if self.arguments.verbose:
-                print("Iteration number {}".format(epoch + 1))
+                print("Iteration number {}".format(1))
```

Comment:
I don't know why you wrote `print("Iteration number {}".format(1))` ? It will be a hardcoded 1 instead of the actual epoch, which is not helpful. Or am I mistaken ?

Reply:
Oh, yeah. It's a mistake. I wanted to write just "epoch" because now it starts from 1.

Reply:
Oh right, it's cleaner indeed then.